server/app/com/xsn/explorer/data/anorm/dao/BlockPostgresDAO.scala

```diff
@@ -307,6 +308,116 @@ class BlockPostgresDAO @Inject()(
     }
   }
 
+  def getBlocks(limit: Limit, orderingCondition: OrderingCondition)(implicit conn: Connection): List[BlockHeader] = {
+    val order = toSQL(orderingCondition)
+
+    val headers = SQL(
+      s"""
+        |SELECT blk.*, tx.count transactions FROM
```

Comment:
When syncing, there might be blocks without transactions, if I understand correctly, it will lead to a NullPointerException as the `tx.count` would be `null`, the trick is `COALESCE(tx.count, 0)` to avoid that issue.